src/main/java/net/dv8tion/jda/core/entities/TextChannel.java

```diff
@@ -87,6 +88,12 @@
      */
     RestAction<Void> deleteMessagesByIds(Collection<String> messageIds);
 
+    RestAction<List<Webhook>> getWebhooks();
+
+    // RestAction<Webhook> createWebhook();
```

Comment:
`createWebhook()` is only in GuildController because it doesn't find a good place in TextChannel.
I at first wanted to place it there but it would collide with the GuildController idea.